src/pages/workspace/expensifyCard/WorkspaceExpensifyCardBankAccounts.tsx

```diff
@@ -37,14 +38,14 @@ type WorkspaceExpensifyCardBankAccountsProps = PlatformStackScreenProps<Settings
 function WorkspaceExpensifyCardBankAccounts({route}: WorkspaceExpensifyCardBankAccountsProps) {
     const {translate} = useLocalize();
     const styles = useThemeStyles();
-    const [bankAccountsList] = useOnyx(ONYXKEYS.BANK_ACCOUNT_LIST);
+    const [bankAccountsList] = useOnyx(ONYXKEYS.BANK_ACCOUNT_LIST, {canBeMissing: false});
 
     const policyID = route?.params?.policyID;
 
     const workspaceAccountID = useWorkspaceAccountID(policyID);
 
-    const [cardBankAccountMetadata] = useOnyx(`${ONYXKEYS.COLLECTION.EXPENSIFY_CARD_BANK_ACCOUNT_METADATA}${workspaceAccountID}`);
-    const [cardOnWaitlist] = useOnyx(`${ONYXKEYS.COLLECTION.NVP_EXPENSIFY_ON_CARD_WAITLIST}${policyID}`);
+    const [cardBankAccountMetadata] = useOnyx(`${ONYXKEYS.COLLECTION.EXPENSIFY_CARD_BANK_ACCOUNT_METADATA}${workspaceAccountID}`, {canBeMissing: false});
```

Comment:
This weird, but `canBeMissing: false` causes this warning everytime I open this page.

![simulator_screenshot_A947DB44-963B-44C4-B1E5-39FBF85D6F34](https://github.com/user-attachments/assets/9e2db029-65d1-478c-b70f-c00b4833d85f)

Reply:
I think we can safely change the flag to true. This should not cause any problems anymore